### PR TITLE
Only damage the first bastion an ender pearl goes through

### PR DESCRIPTION
**Pull request by BlackXnt**
Tested by me and @Maxopoly on test shard.

### CODE REVIEW

**Comment by ProfTANSTAAFL** [comment]
hype

**Comment by ttk2** [comment]
Any reason not to merge?

**Comment by Maxopoly** [comment]
Want confirmation from Dan first that he's on board with this

File: Bastion/src/isaac/bastion/manager/EnderPearlManager.java [hunk 1]
```diff
@@ -453,9 +454,24 @@ public Flight(EnderPearl pearl, long endTime, BastionBlock blocking) {
 		
 		public void cancel() {
 			if (pearl.getShooter() instanceof Player) {
-				handleTeleport(blocking, pearl.getLocation(), (Player) pearl.getShooter());
+				Player player = (Player) pearl.getShooter();
+				
+				if (Bastion.getConfigManager().getDamageFirstBastion() && !Bastion.getBastionManager().onCooldown(player.getName())) {
+					blocking.erode(blocking.erosionFromPearl());
+					pearl.getWorld().spigot().playEffect(pearl.getLocation(), Effect.EXPLOSION, 0, 0, 1, 1, 1, 1, 50, 32);
```
Comment (ProgrammerDan) [inline Bastion/src/isaac/bastion/manager/EnderPearlManager.java]: <3

**Comment by ProgrammerDan** [comment]
Code looks good, hopefully all entry points covered. assuming well tested I have no objection.

**Comment by BlackXnt** [comment]
Good to merge then.

**Pull request merged**